.github/workflows/CD.yml

```diff
@@ -5,13 +5,37 @@ on:
       - v*
 
 jobs:
+  Provision-Runners:
+    name: Provision-Runners
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          repository: opendistro-for-elasticsearch/opendistro-build
+          ref: master
+          path: opendistro-build
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_INFRATEST_EC2_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_INFRATEST_EC2_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+      - name: AWS Cli Processing
+        run: |
+          #!/bin/bash
+          # Please add comma at the end of the RUNNERS strings
+          # Don't add space after commas
+          RUNNERS="odfe-knnlib-build-arm64"
+          export GIT_URL_REPO=${GITHUB_REPOSITORY}
+          ../opendistro-build/release-tools/scripts/setup_runners.sh run $RUNNERS ${{ secrets.ODFE_RELEASE_BOT_PUBLIC_PRIVATE_READ_WRITE_TOKEN }} ami-0ed17173ab64255b1
```

Comment:
From my understanding, AMI Id's can change. How will we know when to change this?

Reply:
The only plugin that uses arm64 alongside infra release workflow is knnlib. All of the images are specifically made for github runners. If we change it on our side we will inform you guys that this value is changed.